libs/langchain/langchain/schema/runnable.py

```diff
@@ -189,6 +190,15 @@ def _call_with_config(
             )
             return output
 
+    def _invoke_with_locals(
+        self,
+        input: Input,
+        _locals: Dict[str, Any],
```

Comment:
if we're trying to avoid shadowing built let's use `_`  as a suffix rather than prefix

```suggestion
        locals_: Dict[str, Any],
```